### PR TITLE
add much dataset

### PR DESCRIPTION
**Pull request by brondle**
Added new datasets for the library with JSON metadata, images coming soon!

### CODE REVIEW

**Review by Erin007 — APPROVED** [review]

**Comment by dancodedotorg** [comment]
:wave: A request: before this gets merged, can we also update the `datasetManifest.js` file to reference these new datasets too? Otherwise they won't show up in the tool. (since this is from Brent's fork, I don't think I can do this myself =/)

**Comment by brondle** [comment]
> > 👋 A request: before this gets merged, can we also update the `datasetManifest.js` file to reference these new datasets too? Otherwise they won't show up in the tool. (since this is from Brent's fork, I don't think I can do this myself =/)
> 
> @brondle Is that something you can pick up?

Sure thing!

**Comment by dancodedotorg** [comment]
@brondle FYI - I snuck in a PR before this one (https://github.com/code-dot-org/ml-playground/pull/92) which also updated the datasetManifest.js file, so you may have to pull my changes first before adding yours.

**Comment by brondle** [comment]
@dancodedotorg @Erin007 added a few more sets and updated the manifest. Erin, I just got the 007 joke in your username 🤦‍♂️

File: src/datasetManifest.js [hunk 1]
```diff
@@ -251,7 +249,85 @@ export const allDatasets = [
     path: "datasets/medical_priority.csv",
     metadataPath: "datasets/medical_priority.json",
     imagePath: "datasets/temp.jpg"
+  },
+  {
+    id: "nutrition",
```
Comment (dancodedotorg) [inline src/datasetManifest.js:254]: @brondle beep boop - looks like nutrition shows up twice? Here and line 261?

Comment (brondle) [reply src/datasetManifest.js:254]: whoops should be fixed

**Pull request merged**